test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/Client/Parallel/ParallelDiscoveryDataAggregatorTests.cs

```diff
@@ -21,13 +20,13 @@ public void AggregateShouldAggregateAbortedCorrectly()
     {
         var aggregator = new ParallelDiscoveryDataAggregator();
 
-        aggregator.Aggregate(totalTests: 5, isAborted: false);
+        aggregator.Aggregate(new(totalTests: 5, isAborted: false));
```

Comment:
I might be too tired today, but what is the change in syntax? is that going from method call with parameters to, method call that takes a single object, which has non-default constructor?

Reply:
The aggregate used to take 2 args (total tests and is aborted), now it takes the event arg (that has an overload with the previous 2 args).